src/client/client.go

```diff
@@ -1413,7 +1414,7 @@ func (s *SignalClient) UpdateProfile(number string, profileName string, base64Av
 		}
```

Comment:
What is missing here is to set the value: e.g: `request.About = about`

Reply:
Added in 57dcb2f
I have not tested this in jsonrpc mode, but it looks like it should work, if I understand the code.